dmscripts/upload_dos_opportunities_email_list.py

```diff
@@ -0,0 +1,46 @@
+from dmscripts.helpers.supplier_data_helpers import SupplierFrameworkData
+
+
+def find_user_emails(supplier_users, services):
+    """Return email addresses for any supplier who has a service in services."""
+    email_addresses = []
+    for service in services:
+        for user in supplier_users.get(service['supplierId'], []):
+            email_addresses.append(user['email address'])
+    return email_addresses
+
+def lowercase_list(list):
```

Comment:
Good point. Have replaced but couldn't think of anything better than a generic `x`